forge.config.ts

```diff
@@ -9,6 +9,17 @@ import { FusesPlugin } from '@electron-forge/plugin-fuses'
 import { FuseV1Options, FuseVersion } from '@electron/fuses'
 import { getPlatform, getArch } from './src/utils/electron'
 
+let resources = [
+  './resources/' + getPlatform() + '/' + getArch(),
```

Comment:
Can we use `path.join('./resources', getPlatform(), getArch())`?

Reply:
makes absolutely more sense :D